forc-plugins/forc-client/src/ops/run/op.rs

```diff
@@ -34,25 +60,18 @@ pub async fn run(command: RunCommand) -> Result<Vec<fuel_tx::Receipt>> {
         .or_else(|| manifest.network.as_ref().map(|nw| &nw.url[..]))
         .unwrap_or(NODE_URL);
     let client = FuelClient::new(node_url)?;
-
-    let build_opts = build_opts_from_cmd(&command);
-    let compiled = match forc_pkg::build_with_options(build_opts)? {
-        pkg::Built::Package(built_package) => *built_package,
-        pkg::Built::Workspace(_) => bail!("running workspaces not yet supported"),
-    };
-
     let contract_ids: Vec<ContractId> = command
         .contract
+        .clone()
```

Comment:
No need to clone the `Vec<String>` here, and we don't want to swallow the error if we can't parse the `ContractId`. Perhaps this could be something like:
```rust
let contract_ids = command
    .as_ref()
    .into_iter()
    .flat_map(|contracts| contracts.iter())
    .map(|contract| ContractId::from_str(&contract))
    .collect::<Result<Vec<ContractId>>>()?;
```